hack/cnao/cnao-cr.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1	
+kind: NetworkAddonsConfig	
+metadata:	
+  name: cluster	
+spec:	
+  imagePullPolicy: Always	
+  linuxBridge: {}	
+  multus: {}	
```

Comment:
Why are you installing linux bridge ? AFAIU, it isn't needed.

Reply:
my bad, will fix